src/main/cpp/Robot.cpp

```diff
@@ -60,6 +61,15 @@ void Robot::RobotInit()
     m_datalogger = DragonDataLoggerMgr::GetInstance();
 
     auto path = AutonUtils::GetPathFromTrajectory("BlueLeftInside_I"); // load choreo library so we don't get loop overruns during autonperiodic
+
+    if (m_dragonswerveposeestimator != nullptr)
+    {
+        auto visionPoseEstitmators = m_dragonswerveposeestimator->GetVisionPoseEstimators();
+        if (!visionPoseEstitmators.empty())
+        {
```

Comment:
Need to add the same check here:
   if (CameraConfigMgr::GetInstance()->GetCurrentConfig()->GetQuestIndex() != -1)